change/@fluentui-react-avatar-0ebe3e03-9d8f-48f7-a4c0-2a95b7014e2e.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "chore: Adding unit tests for AvatarGroup. ",
+  "packageName": "@fluentui/react-avatar",
+  "email": "esteban.230@hotmail.com",
+  "dependentChangeType": "patch"
+}
```

This can be of type none since it is just a change in tests

```suggestion
{
  "type": "none",
  "comment": "chore: Adding unit tests for AvatarGroup. ",
  "packageName": "@fluentui/react-avatar",
  "email": "esteban.230@hotmail.com",
  "dependentChangeType": "none"
}
```